controller/OrderController.go

```diff
@@ -0,0 +1,128 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/dao"
+	_ "github.com/Baraulia/COURIER_SERVICE/dao"
+	"github.com/gin-gonic/gin"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+
+
+     var Orders []dao.Order
+func (h *Handler) GetCourierCompletedOrders(c *gin.Context) {
+	page,er:= strconv.Atoi(c.Query("page"))
+	if er!=nil{
+		c.JSON(http.StatusBadRequest, gin.H{"message": "no such page"})
```

Comment:
better something about the fact that you expect an integer and couldn't parse provided param

Reply:
✅